website/docs/cloud-docs/users-teams-organizations/single-sign-on/entra-id.mdx

```diff
@@ -35,7 +35,7 @@ For more information on the listed features, visit the [Microsoft Entra ID SAML
 
    ![sso-setup](/img/docs/setup.png)
 
-1. Select "Azure" and click "Next".
+1. Select "Microsoft Entra ID" and click "Next".
 
    ![sso-wizard-choose-provider-azure](/img/docs/wizard-choose-provider-azure.png)
```

Comment:
🤔 this screenshot is slightly out of date - is that important? 
Current doc: 
![image](https://github.com/user-attachments/assets/5b7fd68f-322d-4348-b2aa-904d107ac32f)

Current Product:
<img width="600" alt="image" src="https://github.com/user-attachments/assets/15f3afc6-6224-4eb8-bebb-68b47fec4cd8">

Reply:
i've decided to update these screenshots too 🧹